sklearn/cross_validation.py

```diff
@@ -300,7 +307,7 @@ class KFold(_BaseKFold):
 
     See also
     --------
-    StratifiedKFold: take label information into account to avoid building
+    StratifiedKFold take label information into account to avoid building
```

Comment:
No, it's not the colon that messes up the formatting here. Sphinx takes the first word in each line and auto-links it. 
![sphinx](https://cloud.githubusercontent.com/assets/241745/8442834/6d2fd8bc-1f4f-11e5-9171-d4a8ad05b5ae.png)

Use the [numpydoc syntax](https://github.com/numpy/numpydoc/blob/master/numpydoc/docscrape.py#L203): keep the colon, but indent the lines of the description. This should do it.